code/game/machinery/computer/shuttle.dm

```diff
@@ -24,16 +24,16 @@
 		if (istype(W, /obj/item/modular_computer))
 			W = W.GetIdCard()
 		if (!W:access) //no access
-			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough. ")
+			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough.")
 			return
 
 		var/list/cardaccess = W:access
 		if(!istype(cardaccess, /list) || !cardaccess.len) //no access
-			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough. ")
+			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough.")
 			return
 
 		if(!(access_bridge in W:access)) //doesn't have this access
-			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough. ")
+			to_chat(user, "The access level of [W:registered_name]\'s card is not high enough.")
```

Comment:
Understand if you'd rather not, but if you wouldn't mind fixing these lines to remove the `\'` and `:` it would be appreciated.